Source/Core/VideoBackends/Software/SetupUnit.cpp

```diff
@@ -18,6 +18,8 @@ void SetupUnit::Init(OpcodeDecoder::Primitive primitive_type)
   m_VertPointer[1] = &m_Vertices[1];
   m_VertPointer[2] = &m_Vertices[2];
   m_VertWritePointer = m_VertPointer[0];
+
+  m_clipper.Init();
```

Comment:
This will get called a lot more often than `VideoSoftware::Initialize`. Not sure if that really matters though.

Reply:
True, I could just drop this, since we'll have already run this in the clipper instance's constructor.

Reply:
Done